scripts/release-post.js

```diff
@@ -32,16 +32,21 @@ function sendRequest (opts) {
     const options = Object.assign({
       headers: { 'User-Agent': 'nodejs.org release blog post script' }
     }, opts)
-
-    request(options, (err, res, body) => {
+    fetch(options.url, options).then(resp => {
+      if (resp.status !== 200) {
+        return reject(new Error(`Invalid status code (!= 200) while retrieving ${options.url}: ${resp.status}`))
+      }
+      if (options.json) {
+        resp.json().then(json => resolve(json))
+      }
+      resp.text().then(text => resolve(text))
+    }).catch(err => {
+      if (err.status !== 200) {
+        return reject(new Error(`Invalid status code (!= 200) while retrieving ${options.url}: ${err.status}`))
+      }
       if (err) {
         return reject(new Error(`Error requesting URL ${options.url}: ${err.message}`))
```

Comment:
```js
throw new Error(`Error requesting URL ${options.url}: ${err.message}`)
```

and remove the wrapper `if` statement.

Note that `err.message` could be different from current error messages because `err` can be a `FetchError`. If it doesn't break anything I would just remove the whole `catch()` branch instead of using a new `Error`.